kratos/includes/constitutive_law.h

```diff
@@ -588,7 +589,8 @@ class KRATOS_API(KRATOS_CORE) ConstitutiveLaw : public Flags
     /**
      * @brief Adds the initial strain vector if it is defined in the InitialState
      */
-    void AddInitialStrainVectorContribution(Vector& rStrainVector, Parameters& rParameterValues)
+    template<typename TVectorType>
+    void AddInitialStrainVectorContribution(TVectorType& rStrainVector, Parameters& rParameterValues)
```

Comment:
2 questions:
1. can `TVectorType` be `const`?
2. do you need the `Parameters` argument? Seems unused and since it is not a virtual fct I guess you can remove it